src/planning/tests/suggestion-composer-test.ts

```diff
@@ -27,9 +29,16 @@ import {StrategyTestHelper} from '../testing/strategy-test-helper.js';
 // }
 
 describe('suggestion composer', () => {
-  it('singleton suggestion slots', async () => {
+  it('singleton suggestion slots', async function() {
+    if (Flags.withNewStorageStack) {
```

Comment:
FWIW I think Scott is going to get rid of suggestion-composer altogether, so probably not worth fixing these tests at all.